src/todo.ts

```diff
@@ -1,11 +1,13 @@
-﻿interface ListItem {
+﻿// interface that carries attributes about items in the list
+interface ListItem {
     createdAt: Date;
     description: string;
     dueAt: Date;
     id: number;
     isDone: boolean;
 }
 
+// class that is use to create todo lists
```

Comment:
Excellent.  This is a comment in a PR then.  Let''s do a simple exercise here.  Please change this into a simple tsDoc description.  These look something like this:
```
/**
 * I am a brief description of the interface.
 */
```
Do that and push up the commit to this branch so we can see what happens when you do.